editor/editor_test.go

```diff
@@ -503,3 +503,54 @@ func TestEditorShowDecimal(t *testing.T) {
 		t.Errorf("err should be nil but got: %v", err)
 	}
 }
+
+func TestEditorShift(t *testing.T) {
+	ui := newTestUI()
+	editor := NewEditor(ui, window.NewManager(), cmdline.NewCmdline())
+	if err := editor.Init(); err != nil {
+		t.Errorf("err should be nil but got: %v", err)
+	}
+	f, err := ioutil.TempFile("", "bed-test-editor-show-decimal")
+	if err != nil {
+		t.Errorf("err should be nil but got: %v", err)
+	}
+	_, _ = f.WriteString("Hello, world!")
+	if err := f.Close(); err != nil {
+		t.Errorf("err should be nil but got: %v", err)
+	}
+	defer os.Remove(f.Name())
+	if err := editor.Open(f.Name()); err != nil {
+		t.Errorf("err should be nil but got: %v", err)
+	}
+	go func() {
+		for _, e := range []struct {
+			typ   event.Type
+			ch    rune
+			count int64
+		}{
+			{event.ShiftLeft, '<', 1},
+			{event.CursorNext, 'w', 7},
+			{event.ShiftRight, '<', 3},
```

Comment:
It seems that `ch` in the tests is not used but please use `'>'` not to make some confusion.

Reply:
Fixed!

> It seems that `ch` in the tests is not used

I set `event.Event.Rune` because other tests often did so.
I think `ch` is not needed in this situation, but there is no static guarantee.
Hence, I don't mind if you remove `ch` later.